<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSpatialiteSridsDialogBase</class>
 <widget class="QDialog" name="QgsSpatialiteSridsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select a Spatialite Spatial Reference System</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QTreeWidget" name="twSrids">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>SRID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Authority</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Reference Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbnFilter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>26</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonSrid">
        <property name="text">
         <string>SRID</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonName">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>twSrids</tabstop>
  <tabstop>leSearch</tabstop>
  <tabstop>pbnFilter</tabstop>
  <tabstop>radioButtonSrid</tabstop>
  <tabstop>radioButtonName</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsSpatialiteSridsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>QgsSpatialiteSridsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
